app/models/report.rb

```diff
@@ -26,12 +26,11 @@ def created_on
 
   def build_mentions
     active_mentions.destroy_all
-    return unless (report_ids = content.scan(%r{(?<=reports/)\d+}).presence)
+    return unless (report_ids = content.scan(%r{(?<=http://localhost:3000/reports/)\d+}).presence)
```

Comment:
```suggestion
    report_ids = content.scan(%r{(?<=http://localhost:3000/reports/)\d+})
```

そもそもの問題として、0件だからといってreturnしないでいいのでは？
0件だとループを回しても何も起きないので、実質returnしてるのと変わりないことになるはずです

Reply:
僕の前のコメントではpresenceを付けていないのがポイントです。
presenceのありなしで何が変わるのかを確認してみてください〜。